src/applications/applications.controller.ts

```
@@ -70,4 +71,22 @@ export class ApplicationsController {
   ) {
     return this.applicationsService.updateStatus(Number(id), updateStatusDto);
   }
+
+  @Get('/reports/csvexport')
+  @ApiOperation({ summary: 'Export applications as CSV', description: 'Exports applications for a given benefitId and report type as a CSV file.' })
```

ESLint

[error] Replace `·summary:·'Export·applications·as·CSV',·description:·'Exports·applications·for·a·given·benefitId·and·report·type·as·a·CSV·file.'` with `⏎····summary:·'Export·applications·as·CSV',⏎····description:⏎······'Exports·applications·for·a·given·benefitId·and·report·type·as·a·CSV·file.',⏎·`

(prettier/prettier)